### DISCUSSION TITLE
Find window color collision avoidance

### DISCUSSION THREAD

**Discussion by wyager** [helix-editor/helix#10894] [General]
Sometimes when I search, the search highlight color conflicts with the text color.

Would it be possible to check if a character's syntax-highlighted color equals the search highlight color, and if so, change the character's color to something else? 
<img width="853" alt="Screenshot 2024-05-30 at 8 08 25 PM" src="https://github.com/helix-editor/helix/assets/780134/ba2624dc-8743-49ed-8fc5-c6cac4d2c21e">

**Comment by kirawi** [comment]
It's up to the themes to choose what they highlight.

**Comment by wyager** [comment]
This is sort of orthogonal to the theme, is it not? Regardless of theme color choice, it probably shouldn't be possible to have foreground and background be the same color.
